player/public/coreplugins/org.ekstep.pdfrenderer-1.0/renderer/plugin.js

```diff
@@ -471,6 +471,17 @@ org.ekstep.contentrenderer.baseLauncher.extend({
                 }
                 error.logFullError = true;
                 error.message = error.message + "options: " + JSON.stringify(context.optionalData);
+                EventBus.dispatch("renderer:alert:show", undefined, {
+                    title: "Error",
+                    text: "Missing PDF",
+                    type: "error",
+                    data: error,
+                    okBtnText: "Open",
+                    callback : function(){
+                        instance.postError(error);
+                        window.open(pdf_url, "_blank")
```

Comment:
In the callback(error), we are opening the pdf_ulr?
Why is error callback we are opening this?

Reply:
It should do this only for Web. For Mobile it should post message to Mobile client